Source/AzPester.psm1

```diff
@@ -7,11 +7,38 @@ function Invoke-AzPester {
         [String] $Parameters
     )
 
-    $definitionJson = Get-Content $Definition -Raw
+    if ($Definition -NotMatch '\.yaml$' -and $Definition -NotMatch '\.yml$' -and $Definition -NotMatch '\.json$') {
+        Write-Error 'The file type is not supported'
+        Exit
+    }
+
+    [String] $definitionJson = $null
+
+    if ($Definition -match '\.yaml$' -or $Definition -match '\.yml$') {
+        $rawYaml = Get-Content $Definition -Raw
+        # Convert YAML to PowerShell Object
+        $psYaml = (ConvertFrom-Yaml -Yaml $rawYaml)
```

Comment:
where does this `ConvertFrom-Yaml` cmdlet comes from? Seems like I don't have it on my PowerShell installation.

Reply:
I forget to add it to the documentation. It's an external module: https://www.powershellgallery.com/packages/powershell-yaml/0.4.2

Reply:
Doc updated